src/service/profile.js

```diff
@@ -141,7 +141,7 @@ export const setFollow = async (data) => {
   const textTargetUser = `${data.username_follower} started following you.\nSend them a message now`;
   const textOwnUser = `You started following ${data.username_followed}.\nSend them a message now.`;
   const members = [data.user_id_follower, data.user_id_followed];
-  await followClient(members, data, textTargetUser, textOwnUser);
+  followClient(members, data, textTargetUser, textOwnUser);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `await` keyword from the `followClient` function call could potentially lead to race conditions or unexpected behavior. If `followClient` is asynchronous and its result is needed before proceeding, it should be awaited.

```diff
- followClient(members, data, textTargetUser, textOwnUser);
+ await followClient(members, data, textTargetUser, textOwnUser);
```
This change ensures that the execution will pause until `followClient` has finished executing, which can prevent potential issues if the following code depends on the results of `followClient`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->